courierservice/courierservice/urls.py

```diff
@@ -18,4 +18,5 @@
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('', admin.site.urls),
```

Comment:
-fucking shit man what's the fuck nig##r

Reply:
i dont understand u